mailbox/tools/indexer/src/main/java/org/apache/mailbox/tools/indexer/SingleMailboxReindexingTask.java

```diff
@@ -55,22 +59,22 @@ public int getFailedReprocessedMailCount() {
     }
 
     private final ReIndexerPerformer reIndexerPerformer;
-    private final MailboxPath path;
+    private final Mailbox mailbox;
     private final AdditionalInformation additionalInformation;
     private final ReprocessingContext reprocessingContext;
 
     @Inject
-    public SingleMailboxReindexingTask(ReIndexerPerformer reIndexerPerformer, MailboxPath path) {
+    public SingleMailboxReindexingTask(ReIndexerPerformer reIndexerPerformer, Mailbox mailbox) {
```

Comment:
isn't MailboxId supposed to be enough ?

Reply:
I did not wanted to duplicate code between `mailboxPath` & `MailboxId`. This prouved to be convenient.